build_library/dev_container_util.sh

```diff
@@ -6,9 +6,9 @@ get_binhost_url() {
 	local image_group=$1
 	local image_path=$2
 	if [ "${image_group}" == "developer" ]; then
-		echo "https://storage.googleapis.com/flatcar-jenkins/${image_group}/boards/${BOARD}/${FLATCAR_VERSION}/${image_path}"
+		echo "${FLATCAR_DEV_BUILDS}/${image_group}/boards/${BOARD}/${FLATCAR_VERSION}/${image_path}"
 	else
-		echo "https://storage.googleapis.com/flatcar-jenkins/boards/${BOARD}/${FLATCAR_VERSION_ID}/${image_path}"
+		echo "${FLATCAR_DEV_BUILDS}/boards/${BOARD}/${FLATCAR_VERSION_ID}/${image_path}"
 	fi
```

Comment:
Thinking about it, this won't work in the CI. The CI will set `FLATCAR_DEV_BUILDS` to a custom value (the private google bucket) so the official mirror will not be used when the dev container is built via CI.
I'll introduce a command line option to `build_image` to set the dev container's binhost.